src/lib/logical_query_plan/import_node.cpp

```diff
@@ -2,46 +2,60 @@
 
 #include <cstddef>
 #include <memory>
-#include <sstream>
+#include <optional>
 #include <string>
 
+#include <boost/algorithm/string.hpp>
 #include <boost/container_hash/hash.hpp>
 
+#include "magic_enum.hpp"
+
 #include "import_export/file_type.hpp"
 #include "logical_query_plan/abstract_lqp_node.hpp"
 #include "logical_query_plan/abstract_non_query_node.hpp"
+#include "storage/encoding_type.hpp"
 
 namespace hyrise {
 
 ImportNode::ImportNode(const std::string& init_table_name, const std::string& init_file_name,
-                       const FileType init_file_type)
+                       const FileType init_file_type, const std::optional<EncodingType> init_table_encoding)
     : AbstractNonQueryNode(LQPNodeType::Import),
       table_name(init_table_name),
       file_name(init_file_name),
-      file_type(init_file_type) {}
+      file_type(init_file_type),
+      table_encoding(init_table_encoding) {}
```

Comment:
Mini: I guess the optional parameter can be a const ref. Also, we tend to prefer curly braces for initialization.